android-json-form-wizard/src/main/java/com/vijay/jsonwizard/interfaces/JsonApi.java

```diff
@@ -95,7 +99,7 @@ void addOnActivityRequestPermissionResultListener(Integer requestCode,
 
     void setmJSONObject(JSONObject jsonObject);
 
-    void updateGenericPopupSecondaryValues(JSONArray jsonArray);
+    void updateGenericPopupSecondaryValues(JSONArray jsonArray, String stepName);
```

Comment:
This is a public method. Changing method signature is a possible breaking change.

Reply:
This is a major version and changes will likely occur. This has been added to only filter on current step so that forms with multiple steps all the steps are not evaluated when a step is being loaded